aspnetcore/blazor/file-downloads.md

```diff
@@ -102,23 +100,31 @@ async function downloadFileFromStream(fileName, contentStreamReference) {
 function triggerFileDownload(fileName, url) {
   const anchorElement = document.createElement('a');
```

Comment:
@TanayParikh ... Even without link text, is there any indication in the UI that the anchor exists for large file downloads? If so, can this include ...

```javascript
anchorElement.style.display = 'none';
```

Reply:
Honestly, I've never seen the anchor to show up when added like this

Reply:
This shouldn't be an issue as we're just creating the element without actually appending it to the document (ex. `document.body.appendChild`). Hence, it shouldn't show up anywhere.

Reply:
Ok ... thx. I (obviously) haven't had a chance to walk this one (*yet* 😄).